json_serializable/test/src/_json_serializable_test_input.dart

```diff
@@ -398,13 +385,14 @@ enum GoodEnum {
   nullValue
 }
 
+// -
```

Comment:
oops?

Reply:
that was mental note for me it can be removed.

Reply:
done, removed it.